policyengine_us/variables/gov/states/ar/tax/income/exemptions/ar_retirement_or_disability_benefits_exemption_person.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ar_retirement_or_disability_benefits_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Arkansas individual retirement or disability benefits exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=13"
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.irs.income.exemption.traditional_distribution
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        pension_income = (
```

Comment:
Don't need `astype(int)`, and please describe variables if limited to certain eligibility criteria
```suggestion
        pension_income = person("taxable_pension_income", period)
        eligible_pension_income = pension_income * head_or_spouse
```